kolibri/plugins/user/assets/src/views/SignInPage.vue

```diff
@@ -296,7 +296,7 @@
         );
       },
       shouldShowPasswordForm() {
-        return Boolean(this.selectedListUser);
+        return Boolean(this.selectedListUser) && !this.usernameIsInvalid;
```

Comment:
If you click "next" before entering anything and then start typing, it transitions to the password form after 1 character because that's all it takes to make `username` valid.

I'd suggest also changing `setSelectedListUser` to not save the `selectedListUser` if there is no `username` given.

Reply:
@nucleogenesis Done